src/gui/dialogs/multiplayer/mp_alerts_options.cpp

```diff
@@ -34,6 +34,19 @@
 namespace gui2::dialogs
 {
 
+const std::vector<std::string> items{
```

Comment:
Since the namespace doesn't give context anymore for this variable a better name would be good. Perhaps `mp_ui_alerts`.

Reply:
Well, it doesn't exist anymore either.

Reply:
I was looking at commit-by-commit. Is the intention to squash this PR?

Reply:
IMO at least the reorganization of the MP alerts options and preferences and the removal of the editor_default_dir preference should be their own commits.

Reply:
I will try to clean it up like that. Previous times when I've tried to move around commits that touch the same code in a PR though, I've just made a mess of things.